packages/@o3r/rules-engine/src/fact/fact.spec.ts

```diff
@@ -40,6 +47,7 @@ describe('Rules engine fact', () => {
     subjectFact.next('test4');
     await jest.runAllTimersAsync();
 
+    // eslint-disable-next-line @typescript-eslint/unbound-method -- method is not called here, we use reference for the mock engine
```

Comment:
this is part of `jest/unbound-method` rule

Reply:
Will not be raised anymore after rebase